app/classes/Keyboard.js

```diff
@@ -21,6 +21,10 @@ export default class Keyboard {
         return this.keys()[index]
     }
 
+    refreshKeys(){
```

Comment:
i wonder if we need to delete the synths when we refresh. Could that be the basis for the new synth bug?